src/report/report_utils.py

```diff
@@ -0,0 +1,498 @@
+"""
+Module containing utility functions used during report generation.
+"""
+
+
+from subprocess import check_output
+from os import environ, getcwd
+from os.path import splitext, basename, join
+
+from docx import Document
+from docx.text.paragraph import Paragraph
+from docx.table import Table, _Cell
+from docx.enum import table, text
+from docx.shared import Emu, Pt, RGBColor
+from docx.oxml.shared import qn, OxmlElement
+
+
+LIBREOFFICE = environ.get("LIBREOFFICE", "")
+if not LIBREOFFICE:
+    print("Environmental variable LIBREOFFICE not set. PDF convertion won't function!")
+    print("LIBREOFFICE should be set to path to soffice/soffice.exe included with LibreOffice installation")
+
+
+def create_stylised_document() -> Document:
+    """
+    Creates document and sets all styles for headings and tables.
+
+    Returns:
+        Reference to created document.
```

Comment:
I think in Python every object is a reference, so I'd change this to just "The created document".
Same thing with `create_company_header()`, `create_table_of_contents()` and `create_table()`